src/test/java/com/puppycrawl/tools/checkstyle/internal/CliOptionsXdocsSyncTest.java

```diff
@@ -63,24 +59,24 @@ public void validateCliDocSections() throws Exception {
         for (OptionSpec opt : optionSpecList) {
             final String option = opt.names()[0];
             if ("-h".equals(option) || "-V".equals(option)) {
-                cmdDesc.remove(option);
+                cmdOptions.remove(option);
                 continue;
             }
-            final String descXdoc = cmdDesc.get(option);
+            final String descXdoc = cmdOptions.get(option);
             final String descMain = opt.description()[0];
             assertWithMessage("CLI Option: " + option + " present in "
                     + "Main.java but not documented in cmdline.xml.vm")
                     .that(descXdoc)
                     .isNotNull();
             assertWithMessage("CLI options descriptions in xdoc: "
                     + " should match that of in Main.java")
-                .that(descMain)
-                .isEqualTo(descXdoc);
-            cmdDesc.remove(option);
+                    .that(descMain)
+                    .isEqualTo(descXdoc);
+            cmdOptions.remove(option);
         }
         assertWithMessage("CLI Options: %s present in cmdline.xml.vm, not documented in Main.java",
-                    cmdDesc)
-                .that(cmdDesc)
+                cmdOptions.values())
+                .that(cmdOptions.values())
```

Comment:
Actually this should be either
```suggestion
                cmdOptions)
                .that(cmdOptions)
```
or
```suggestion
                cmdOptions.keys())
                .that(cmdOptions.keys())
```
No real preference

Reply:
Let's use the first option 'that(cmdOptions)'

Reply:
Updated

Reply:
@kkoutsilis , please do not resolve conversations.
Please allow reviewer to confirm that all is done as required and threre is no misunderstndings.